src/script/components/CopyToClipboard.tsx

```diff
@@ -19,24 +19,30 @@
 
 import React from 'react';
 import {registerReactComponent} from 'Util/ComponentUtil';
+import {handleKeyDown} from 'Util/KeyboardUtil';
 
 export interface CopyToClipboardProps {
   text: string;
 }
 
 const CopyToClipboard: React.FC<CopyToClipboardProps> = ({text}) => {
+  const onClick = ({currentTarget}: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+    if (window.getSelection) {
+      const selectionRange = document.createRange();
+      selectionRange.selectNode(currentTarget);
+      window.getSelection().removeAllRanges();
+      window.getSelection().addRange(selectionRange);
+    }
+  };
+
   return (
     <div
+      role="button"
+      tabIndex={0}
       data-uie-name="copy-to-clipboard"
       className="copy-to-clipboard"
-      onClick={({currentTarget}) => {
-        if (window.getSelection) {
-          const selectionRange = document.createRange();
-          selectionRange.selectNode(currentTarget);
-          window.getSelection().removeAllRanges();
-          window.getSelection().addRange(selectionRange);
-        }
-      }}
+      onClick={onClick}
+      onKeyDown={e => handleKeyDown(e, onClick.bind(null, e))}
```

Comment:
```suggestion
      onKeyDown={e => handleKeyDown(e, onClick)}
```
I've been doing this, any reason that wouldn't work?

Reply:
onClick function requires `event` as an argument

Reply:
If it was param-less function we could include it as you suggested